### PR TITLE
Simplify console re-run with rich Confirm

### PR DESCRIPTION
**Pull request by ehsanmok**

### CODE REVIEW

**Review by alexandrnikitin — APPROVED** [review]

File: examples/console.py [hunk 1]
```diff
@@ -20,17 +20,18 @@
 try:
     from rich.console import Console
     from rich.table import Table
+    from rich.prompt import Confirm
```
Comment (tzhenghao) [inline examples/console.py:23]: nit: `import` in alphabetical order

File: examples/console.py [hunk 2]
```diff
@@ -20,17 +20,18 @@
...
 
 except ImportError:
     print("rich not found. Installing rich...")
     subprocess.run(["python3", "-m", "pip", "install", "rich"])
     from rich.console import Console
     from rich.table import Table
+    from rich.prompt import Confirm
```
Comment (tzhenghao) [inline examples/console.py:30]: ditto

**Review by tzhenghao — APPROVED** [review]
LGTM! Left a nit comment

**Comment by tzhenghao** [comment]
Merging this for you @ehsanmok since we wanna close out remaining PRs

**Pull request merged**